### DISCUSSION TITLE
1.8.0 - Issues with displayLegend

### DISCUSSION THREAD

**Discussion by yosland9** [MMRIZE/MMM-CalendarExt3#122] [Q&A] [answered]
Hi there - first time posting and not really a developer - hopefully you can bear with me -  I have been running MMM-CalendarExt3 and Agenda perfectly well for months.  When I updated to 1.8.0, Agenda still worked, but I was getting an "UNDEFINED" quick pop-up on launch of MM, and CalExt3 did not display.  Looking at the logs, I see if fetched the calendars, etc, but nothing displayed.  After digging through the npm run start:dev, I saw there were error related to "events" not being defined and it referenced the legend section.  When I commented out the displayLegend = true, I began to see CalExt3 launch and be visible.   I can get more details in a day or so of the errors, but I was hoping to see if anyone else had this issue?  (all the cal set names are defined in the Calendar module).

**Comment by eouia** [comment, marked as answer]
Sorry. I update the fix.
